common/djangoapps/student/views/dashboard.py

```diff
@@ -194,6 +198,21 @@ def _create_recent_enrollment_message(course_enrollments, course_modes):  # pyli
             }
         )
 
+def get_course_prices(enrollments):
+    """
+    Given a list of enrollments return a dict of course ids with their prices.
+
+    Utility function for experimental metadata. See experiments/dashboard_metadata.html.
+
+    :param enrollments:
+    :return: dict of courses: course price for dashboard metadata
+    """
+    if dashboard_metadata_flag.is_enabled():
+        course_prices = {}
+        for enrollment in enrollments:
+            course_prices[enrollment.course] = enrollment.course_price
+        return course_prices
+
```

Comment:
Do we need to return anything if the flag is not enabled?

Reply:
It'll automatically return None, but I can make that more explicit if you think that's warranted?